### ISSUE TITLE
Moving an existing item "into the scope" should add it to the bottom.

### ISSUE THREAD

**Issue by manuelmeurer** [pixeltrix/sortifiable#1]
Example:

```
class List < AR::Base
  has_many :items
end

class Item < AR::Base
  belongs_to :list

  acts_as_list, :scope => :list
end

# Existing lists:
# #<List id: 1>
# #<List id: 2>

# Existing items:
# #<Item id: 1, list_id: 1, position: 1>
# #<Item id: 2, list_id: 1, position: 2>
# #<Item id: 3, list_id: 2, position: 1>

item = Item.find(3)
item.list = List.find(1)
item.save!
```

I would expect:

```
#<Item id: 1, list_id: 1, position: 1>
#<Item id: 2, list_id: 1, position: 2>
#<Item id: 3, list_id: 1, position: 3>
```

Instead:

```
#<Item id: 1, list_id: 1, position: 1>
#<Item id: 2, list_id: 1, position: 2>
#<Item id: 3, list_id: 1, position: 1>
```

Same with moving an item out of the list, I would expect the lower items to "close the gap".
Do you agree?
If yes, and if you would accept a pull request, I would be willing to work on that.

**Comment by pixeltrix** [comment]
I agree - however detecting when the item has moved in or out of the scope may be tricky. More than willing to accept a pull request.

**Comment by manuelmeurer** [comment]
Ok, I'll see if I can make it work.

**Comment by pixeltrix** [comment]
Fixed in gem release 0.2.1 - thanks for the patch
